scheduling/src/waypoint_scheduler.cpp

```diff
@@ -109,6 +109,7 @@ std::vector<scheduling::Action> scheduling::WaypointScheduler::convertResult(
     std::vector<scheduling::Action> schedule;
     cur_waypoint.pop();
     scheduling::TimeValuePair last_cur = cur_waypoint.front();
+    dest_waypoint.pop();
```

Comment:
Tænker ikke at det her er en del af at bruge regex, så hvad er det for en ændring?

Reply:
Det handler om fortolkning af det parsede data.